backend/api/user_accepted_requests.go

```diff
@@ -0,0 +1,32 @@
+package api
+
+import (
+	"database/sql"
+	"net/http"
+
+	db "github.com/git-adithyanair/cs130-group-project/db/sqlc"
+
+	"github.com/gin-gonic/gin"
+	"github.com/git-adithyanair/cs130-group-project/token"
+)
+
+func (server *Server) GetUserAcceptedRequest(ctx *gin.Context) {
+
+	authPayload := ctx.MustGet(authorizationPayloadKey).(*token.Payload)
+
+	arg := db.GetRequestsForUserByStatusParams{
+		UserID: authPayload.UserID,
+		Status: db.RequestStatusInProgress,
```

Comment:
I could potentially change this API to be more flexible and get requests for user by status if that would be needed

Reply:
Actually yeah, that's a good idea. I'm thinking we can just make this endpoint to retrieve all requests for the user and separate them by status. So the response would be something like `{accepted_requests: [...], pending_requests: [...], ...}`

Reply:
oh yes that is a good idea, will do